probe_test.go

```diff
@@ -728,6 +728,63 @@ func TestLinkStatusUnknown(t *testing.T) {
 	}
 }
 
+func TestVirtualWanHealhCheck(t *testing.T) {
+	c := newFakeClient()
+	c.prepare("api/v2/monitor/virtual-wan/health-check", "testdata/virtual_wan_health_check.jsonnet")
+	r := prometheus.NewPedanticRegistry()
+	if !testProbe(probeVirtualWanPerf, c, r) {
+		t.Errorf("probeVirtualWanPerf() returned non-success")
+	}
+
+	em := `
+		# HELP fortigate_virtual_wan_healthcheck_active_sessions Active Session count for the health check interface
+		# TYPE fortigate_virtual_wan_healthcheck_active_sessions gauge
+		fortigate_virtual_wan_healthcheck_active_sessions{interface="WAN1_VL300",sla="root",vdom="Internet Check"} 710
```

Comment:
Seems VDOM and SLA have been mixed here.

Reply:
Thanks, Good Find! I mixed the variable order.